lib/dl_connector_bundle_chs3/dl_connector_bundle_chs3/chs3_base/core/data_source.py

```diff
@@ -102,7 +102,6 @@ def spec(self) -> BaseFileS3DataSourceSpec:
 
     def get_parameters(self) -> dict:
         return dict(
-            super().get_parameters(),
```

Comment:
Here is the fix for source replacement.

When you replace a connection in a dataset (which completes with an error, because there is no custom migration defined) and go for the source replacement, control-api tries to compare data source parameters between the old version and the new one [here](https://github.com/datalens-tech/datalens-backend/blob/f16ecf075b3b84ed6708f77090f1146d8e62e287/lib/dl_api_lib/dl_api_lib/api_common/dataset_loader.py#L151-L151), and when it comes to this line it then [tries](https://github.com/datalens-tech/datalens-backend/blob/f16ecf075b3b84ed6708f77090f1146d8e62e287/lib/dl_core/dl_core/data_source/sql.py#L316-L316) to fetch the old connection attributes, but that connection has no reason to be loaded in the US entry buffer => `ValueError: Connection DefaultConnectionRef(conn_id='xxxxxxxxxxxxx') is not loaded` exception is raised.

In reality the only applicable parameter for file sources is the `origin_source_id`.